include/deal.II/grid/grid_generator.h

```diff
@@ -135,6 +135,16 @@ namespace GridGenerator
    * <tt>Triangulation@<2,3@></tt> will be a square in the xy plane with z=0.
    *
    * @note The triangulation passed as argument needs to be empty when calling this function.
+   *
+   * @param tria The Triangulation to create. It needs to be empty upon
+   * calling this function.
+   *
+   * @param repetitions A vector of @p dim positive values denoting the number
+   * of cells to generate in that direction.
```

Comment:
Could we add one more sentence here that clarifies that the first direction is `x`, the second is `y`, etc.?